### PR TITLE
Remove explicit protocol mentions from scripts

### PR DESCRIPTION
**Pull request by haarts**
See the commit message for additional information!

I'd be willing to investigate if the `scripts/util.py` module can be removed in favor of the plain network handling as is done in `scripts/watch_address` for example. 

You'll be seeing more of these small PRs from me soon, I've opted to keep them small in order to give you an opportunity to comment often.

### CODE REVIEW

File: lib/network.py [hunk 1]
```diff
@@ -1076,13 +1077,45 @@ def synchronous_get(self, request, timeout=30):
...
+
+    def subscribe_to_headers(self):
+        command = 'blockchain.headers.subscribe'
+        return (command, [])
+
+    def subscribe_to_address(self, address):
+        command = 'blockchain.address.subscribe'
+        return (command, [address])
+
+    def get_merkle_for_transaction(transaction_hash, transaction_height):
```
Comment (SomberNight) [inline lib/network.py]: what about `self`? or maybe you want this to be a `staticmethod`?

File: lib/network.py [hunk 2]
```diff
@@ -1076,13 +1077,45 @@ def synchronous_get(self, request, timeout=30):
...
+        command = 'blockchain.transaction.get_merkle'
+        return (command, [transaction_hash, transaction_height])
+
+    def subscribe_to_scripthash(scripthash):
```
Comment (SomberNight) [inline lib/network.py]: `self` again

File: lib/network.py [hunk 3]
```diff
@@ -1076,13 +1077,45 @@ def synchronous_get(self, request, timeout=30):
...
+        command = 'blockchain.scripthash.subscribe'
+        return (command, [scripthash])
+
+    def get_transaction(transaction_hash):
```
Comment (SomberNight) [inline lib/network.py]: `self` again

File: lib/network.py [hunk 4]
```diff
@@ -1076,13 +1077,45 @@ def synchronous_get(self, request, timeout=30):
...
+        command = 'blockchain.transaction.get'
+        return (command, [transaction_hash])
+
+    def listunspent_for_scripthash(scripthash):
```
Comment (SomberNight) [inline lib/network.py]: `self` again

File: lib/network.py [hunk 5]
```diff
@@ -1076,13 +1077,45 @@ def synchronous_get(self, request, timeout=30):
...
+        command = 'blockchain.scripthash.listunspent'
+        return (command, [scripthash])
+
+    def get_balance_for_scripthash(scripthash):
```
Comment (SomberNight) [inline lib/network.py]: `self` again

File: lib/network.py (hunk 1 above)
Comment (haarts) [reply lib/network.py]: You are 100% right. I realized that when my work progressed and I've addressed that in #4376, https://github.com/spesmilo/electrum/pull/4376/files#diff-08f47fc4236c4931079fa1058b996bb2R1128

**Comment by SomberNight** [comment]
Oh wait, so https://github.com/spesmilo/electrum/pull/4376 includes all these commits?
Then there is no reason for this PR anymore.

**Pull request closed without merging**